index.js

```diff
@@ -0,0 +1,132 @@
+  const deleteButtons = document.querySelectorAll('.delete-btn');
+  deleteButtons.forEach((button) => {
+    button.onclick = () => {
+      const { dataset: { bookIndex } } = button;
+      const targetBook = document.querySelector(`li[data-book-index="${bookIndex}"]`);
+      removeBook(bookIndex);
+      inventory.removeChild(targetBook);
+    };
+  });
+}
+
+function createInventory(library) {
+  inventory.innerHTML = '';
+  library.forEach((book) => {
+    const li = document.createElement('li');
+    li.className = 'column is-full-mobile is-two-thirds-tablet is-half-desktop is-one-third-widescreen is-one-quarter-fullhd';
+    const bookIndex = library.indexOf(book);
+    li.setAttribute('data-book-index', `${bookIndex}`);
+    const statusColor = book.status === 'Already read!' ? 'is-success' : '';
+    const bookContent = `<div class="card">
+                          <div class="card-image">
+                            <figure class="image is-4by3">
+                              <img src="https://source.unsplash.com/t7zYZzO_CX0/1600x900" alt="Book cover">
+                            </figure>
+                          </div>
+                          <div class="card-content">
+                            <div class="media">
+                              <div class="media-left">
+                                <figure class="image is-48x48">
+                                  <img src="https://source.unsplash.com/-FVaZbu6ZAE/1600x900" alt="Book thumbnail">
+                                </figure>
+                              </div>
+                              <div class="media-content">
+                                <p class="title is-4">${book.title}</p>
+                                <p class="subtitle is-6">${book.author}</p>
+                              </div>
+                            </div>
+                            <div class="content">
+                              Number of pages:
+                              <br>
+                              ${book.pages}
+                            </div>
+                          </div>
+                          <footer class="card-footer">
+                            <button data-book-index="${bookIndex}" class="button ${statusColor} change-btn card-footer-item">${book.status}</button>
+                            <button data-book-index="${bookIndex}" class="button is-danger delete-btn card-footer-item" >Delete</button>
+                          </footer>
+                        </div>`;
+    li.insertAdjacentHTML('beforeend', bookContent);
+    inventory.appendChild(li);
+  });
+  addChangeListeners();
+  addDeleteListeners();
+}
+
+function addBookToLibrary() {
+  const title = document.querySelector('[name="book-title"]').value;
+  const author = document.querySelector('[name="book-author"]').value;
+  const pages = document.querySelector('[name="book-pages"]').value;
+  const status = document.querySelector('[name="book-status"]').checked;
```

Comment:
As outlined by the previous TSE, I was able to create an empty data, which is not allowed. Let us implement validations for adding book; eg. 
```
function addBookToLibrary(title, author, pages) {
  title = document.querySelector('[name="book-title"]').value;
  author = document.querySelector('[name="book-author"]').value;
  pages = document.querySelector('[name="book-pages"]').value;
if (title && author && pages) {
  status = document.querySelector('[name="book-status"]').checked;
  const book = new Book(title, author, pages, status);
  myLibrary.push(book);
  createInventory(myLibrary);
  resetForm();
  storeLibrary();
} else {
error_msg();
}

}
```
Something similar to this would be great